### PR TITLE
Allow customization of ports and names

### PR DESCRIPTION
**Pull request by serpi90**

### CODE REVIEW

File: local-build.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```
Comment (gcotelli) [inline local-build.sh]:
```suggestion
#!/usr/bin/env bash
```

**Review by gcotelli — APPROVED** [review]
Just a little suggestion, the rest LGTM

**Review by jvanecek — APPROVED** [review]

**Comment by fortizpenaloza** [comment]
Why did you left the branch? @serpi90

**Comment by serpi90** [comment]
forgot to remove it

**Pull request merged**